### PR TITLE
Service plus custom

### PR DESCRIPTION
**Pull request by kjain110**
## Purpose
_Describe the purpose of the pull request. Include background information if necessary._

## Approach
_How does this change fulfill the purpose?_

#### TODOS and Open Questions
- [ ] Use GitHub checklists. When solved, check the box and explain the answer.

## Learning
_Describe the research stage. Add links to blog posts, patterns, libraries or addons used to solve this problem._

## Pre-Merge Checklist

If you are adding entity type(s), have you:
- [ ] Added the JSON5 definition to the `src/main/resources/entity-types` directory?
- [ ] Ensured that GETing the entity type at `/entity-types/{id}` works as expected?
- [ ] Added translations for all fields, per the [translation guidelines](/translations/README.md)? (Check this by ensuring `GET /entity-types/{id}` does not have `mod-fqm-manager.entityType.` in the response)
- [ ] Added views to liquibase, as applicable?
- [ ] Added required interfaces to the module descriptor?
- [ ] Checked that querying fields works correctly and all SQL is valid?

### CODE REVIEW

File: src/main/resources/entity-types/inventory/drv_service_point_details.json5 [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->'metadata'->>'updatedDate'",
+    },
+    {
+      name: 'created_by_user_id',
+      sourceAlias: 'service_point',
+      dataType: {
+        dataType: 'stringType',
```
Comment (ncovercash) [inline src/main/resources/entity-types/inventory/drv_service_point_details.json5:73]: Shouldn't this (and updated) be uuid?

File: src/main/resources/entity-types/users/drv_custom_field_details.json5 [hunk 2]
```diff
@@ -0,0 +1,344 @@
+{
+  id: '0b70ae1e-a87a-4621-a2d8-143973821e39',
+  name: 'drv_custom_field_details.json5',
```
Comment (ncovercash) [inline src/main/resources/entity-types/users/drv_custom_field_details.json5:3]: json5 shouldn't be in the name here (also maybe add user, eg drv_user_custom_field_details)?

File: translations/mod-fqm-manager/en.json [hunk 3]
```diff
@@ -71,6 +90,34 @@
   "entityType.drv_contributor_type_details.updated_by_user_id": "Updated by user ID",
   "entityType.drv_contributor_type_details.updated_by_username": "Updated by username",
   "entityType.drv_contributor_type_details.updated_date": "Updated date",
+  "entityType.drv_custom_field_details.json5": "Custom field",
```
Comment (ncovercash) [inline translations/mod-fqm-manager/en.json:93]: Specify this is user's?

File: translations/mod-fqm-manager/en.json [hunk 4]
```diff
@@ -1,4 +1,23 @@
 {
+  "entityType..code": "Code",
```
Comment (ncovercash) [inline translations/mod-fqm-manager/en.json:2]: These should be removed (I'm guessing these were for service points but you hadn't named the entity type yet)

File: translations/mod-fqm-manager/en.json [hunk 5]
```diff
@@ -71,6 +90,34 @@
...
   "entityType.drv_contributor_type_details.updated_by_username": "Updated by username",
   "entityType.drv_contributor_type_details.updated_date": "Updated date",
+  "entityType.drv_custom_field_details.json5": "Custom field",
+  "entityType.drv_custom_field_details.json5.checkbox_field_default": "Checkbox field default",
+  "entityType.drv_custom_field_details.json5.created_by_user_id": "Created by user ID",
+  "entityType.drv_custom_field_details.json5.created_by_username": "Created by username",
+  "entityType.drv_custom_field_details.json5.created_date": "Created date",
+  "entityType.drv_custom_field_details.json5.entity_type": "Entity type",
+  "entityType.drv_custom_field_details.json5.help_text": "Help text",
+  "entityType.drv_custom_field_details.json5.id": "ID",
```
Comment (ncovercash) [inline translations/mod-fqm-manager/en.json:100]: UUID

File: translations/mod-fqm-manager/en.json [hunk 6]
```diff
@@ -71,6 +90,34 @@
   "entityType.drv_contributor_type_details.updated_by_user_id": "Updated by user ID",
   "entityType.drv_contributor_type_details.updated_by_username": "Updated by username",
   "entityType.drv_contributor_type_details.updated_date": "Updated date",
+  "entityType.drv_custom_field_details.json5": "Custom field",
+  "entityType.drv_custom_field_details.json5.checkbox_field_default": "Checkbox field default",
+  "entityType.drv_custom_field_details.json5.created_by_user_id": "Created by user ID",
```
Comment (ncovercash) [inline translations/mod-fqm-manager/en.json:95]: UUID (ditto for several others)

**Review by ncovercash — CHANGES_REQUESTED** [review]

File: translations/mod-fqm-manager/en.json (hunk 5 above)
Comment (kjain110) [reply translations/mod-fqm-manager/en.json]: @ncovercash  wym ?

**Comment by kjain110** [comment]
@ncovercash  will change to UUID in another PR

**Review by mweaver-ebsco — APPROVED** [review]

File: src/main/resources/entity-types/users/drv_custom_field_details.json5 [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+    {
+      name: 'order',
+      sourceAlias: 'custom_fields',
+      dataType: {
+        dataType: 'integerType',
+      },
+      isIdColumn: false,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'order'",
```
Comment (bvsharp) [inline src/main/resources/entity-types/users/drv_custom_field_details.json5:56]: we need to cast this to an integer here

File: src/main/resources/entity-types/inventory/drv_service_point_details.json5 [hunk 8]
```diff
@@ -0,0 +1,229 @@
+{
+  id: '1fdcc2e8-1ff8-4a99-b4ad-7d6bf564aec5',
+  name: 'drv_service_point_detail',
```
Comment (bvsharp) [inline src/main/resources/entity-types/inventory/drv_service_point_details.json5:3]: nit: could call this drv_service_point_details to match the filename

**Review by bvsharp — APPROVED** [review]

**Pull request merged**